src/Data.jsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 const Data = ({ data, onChange }) => (
   <div>
-    <textarea className="data" onChange={onChange}>
+    <textarea id="dataTextarea" className="data" onChange={onChange}>

```

Comment:
What's the reason for needing the id? (Just curious, not suggesting you should change it or anything...)

Reply:
It was a strange one, so when you click on the example JSON button it updated the state value to the new values allowing it to be presented correctly in the preview, but the data text area's value was [object, Object]. To get around this I also update the value of the text area itself by using the id of the textarea. This may be a bit of a hacky solution but I was going around in circles and this was the only way I got it to work properly 😂

Reply:
Weird! Well, if it works, that's the main thing! One of these days we'll bring this "product" in-house and at that point we might fix oddities like this, but it can stay odd for the time being ;-)